### DISCUSSION TITLE
Certificates for installation of Yarra RDS on an XA30 scanner.

### DISCUSSION THREAD

**Discussion by RFranklin-MR** [yarra-framework/yarra-framework.github.io#5] [Q&A] [answered]
Hi Yarra team,

Hopefully this is the correct way to contact you regarding the certificates that are needed to install Yarra RDS client on an XA Siemens system. How would I go about getting a copy of the certificates and installation instructions?

Many thanks,
Rachael

**Comment by tblock79** [comment, marked as answer]
Please send your email address to tobias.block@nyumc.org, and I will send you the certificate.

Best,
  Tobias
